### ISSUE TITLE
controller installation: dracut error installing lspci

### ISSUE THREAD

**Issue by themisa** [clustervision/trinity#242]
**Software:** master_installer 6739973
**Component:** install controller
**Hardware:** SM controller02
**Test case:** 1.1.3
**Bug description:**
```
dracut-install: ERROR: installing 'lspci'
dracut-install: ERROR: installing 'setpci'
```
**Steps to reproduce:**
install controller
 cat /var/log/xcat/xcat.log | grep -i "error"

**Expected output:**
no error

**Comment by hansthen** [comment]
Apart from the error message, is there anything that appears not to work because of this error message?

**Comment by themisa** [comment]
To avoid error add pciutils to the packagelist.

**Comment by linuxstudio** [comment]
I applied in dtap the same fix that hansthen applied for r3, i.e. adding pciutils to controller/rootimg/install/custom/netboot/centos/trinity.centos7.pkglist. But this doesn't seem to fix (completely) the problem.

Before applying the fix I had:

    [root@controller ~]# cat /var/log/xcat/xcat.log | grep -i "error"
    dracut-install: ERROR: installing 'lspci'
    dracut-install: ERROR: installing 'setpci'
    /usr/lib/dracut/dracut-install -D /var/tmp/initramfs.3OiK7I -a lspci setpci awk sleep
    dracut-install: ERROR: installing '/sbin/lspci'
    /usr/lib/dracut/dracut-install -D /var/tmp/initramfs.9MnOlr /sbin/lspci
    dracut-install: ERROR: installing '/usr/sbin/dmidecode'
    /usr/lib/dracut/dracut-install -D /var/tmp/initramfs.9MnOlr /usr/sbin/dmidecode
    dracut-install: ERROR: installing '/sbin/lspci'
    dracut-install: ERROR: installing '/usr/sbin/dmidecode'

After commit 027672f I have:

    [root@controller ~]# cat /var/log/xcat/xcat.log | grep -i "error"
    dracut-install: ERROR: installing 'lspci'
    dracut-install: ERROR: installing 'setpci'
    /usr/lib/dracut/dracut-install -D /var/tmp/initramfs.KzaK8S -a lspci setpci awk sleep
    dracut-install: ERROR: installing '/usr/sbin/dmidecode'
    dracut-install: ERROR: installing '/usr/sbin/dmidecode'
    /usr/lib/dracut/dracut-install -D /var/tmp/initramfs.EXlEsT /usr/sbin/dmidecode
    
    [root@controller ~]# which lspci
    /usr/bin/which: no lspci in (/opt/xcat/bin:/opt/xcat/sbin:/opt/xcat/share/xcat/tools:/usr/local/sbin:    /usr/local/bin:/usr/sbin:/usr/bin:/root/bin)
    
    [root@controller ~]# yum provides lspci
    pciutils-3.2.1-4.el7.x86_64 : PCI bus related utilities
    Repo        : local-centos7.0-x86_64-x86_64
    Matched from:
    Filename    : /usr/sbin/lspci
    
    [root@controller ~]# which dmidecode
    /usr/sbin/dmidecode
    
    [root@controller ~]# yum provides dmidecode
    1:dmidecode-2.12-5.el7.x86_64 : Tool to analyse BIOS DMI data
    Repo        : local-centos7.0-x86_64-x86_64


So seems that we have still two issues:

1) pciutils is not installed on the controller (while it should be) 
2) installation of pciutils in trinity.centos7 is not enoug: we need also dmidecode (which is available on the controller)

**Comment by linuxstudio** [comment]
After commit 504e1c0 we still have one pci-related error:

    [root@controller ~]# cat /var/log/xcat/xcat.log | grep -i "error"
    ...
    dracut-install: ERROR: installing 'lspci'
    dracut-install: ERROR: installing 'setpci'
    /usr/lib/dracut/dracut-install -D /var/tmp/initramfs.PGKntz -a lspci setpci awk sleep

**Comment by linuxstudio** [comment]
Notice that some changes need to be merged back to r3

**Comment by themisa** [comment]
fixed for r3 7ac7a06
